Ginger/GingerCore/ALM/JIRA/Bll/JiraExportManager.cs

```diff
@@ -47,15 +47,85 @@ public JiraExportManager(JiraRepository.JiraRepository jiraRep)
         {
             Dictionary<Guid, string> defectsOpeningResults = new Dictionary<Guid, string>();
             List<JiraIssueExport> defectsToExport = new List<JiraIssueExport>();
+            List<string> screenshots = new List<string>();
             foreach (KeyValuePair<Guid, Dictionary<string, string>> defectForOpening in defectsForOpening)
             {
+                string summaryValue = defectForOpening.Value.ContainsKey("Summary") ? defectForOpening.Value["Summary"] : string.Empty;
+
+                if (!string.IsNullOrEmpty(summaryValue))
+                {
+                    //Search for exist based on summary
+                    WhereDataList filterData = new WhereDataList();
+                    filterData.Add(new WhereData() { Name = "summary", Values = new List<string>() { summaryValue }, Operator = WhereOperator.And });
+
+                    //and based on status, fetch statuses from JiraSettings.json 
+                    DefectStatusColl jiraDefectStatuses = null;
+                    JiraRepository.Helpers.JiraHelper jiraHelper = new JiraRepository.Helpers.JiraHelper();
+                    if (jiraHelper.TryGetJiraDefectStatuses(out jiraDefectStatuses))
+                    {
+                        foreach (DefectStatus defectStatus in jiraDefectStatuses)
+                        {
+                            ComparisionOperator comparisionOperator = ComparisionOperator.EqualTo;
+                            if (defectStatus.Operator.ToLower().Contains("not"))
```

Comment:
Please add Switch case and return the operator based on matching string instead of contains. We might add more operators in future